glue/python/src/stim/__init__.pyi

```diff
@@ -4378,6 +4378,48 @@ class PauliString:
     @sign.setter
     def sign(self, value: complex):
         pass
+    def to_tableau(
+        self,
+    ) -> stim.Tableau:
+        """Creates a Tableau equivalent to this Pauli string.
+
```

Comment:
Add a note that the global phase of the pauli string is lost. Both +P and -P and i*P all map to the same tableau.

Reply:
:+1: